test/doc/core_4_layers.cpp

```diff
@@ -265,7 +265,7 @@ class counted_stream
     /// Write some data to the stream asynchronously
     template<
         class ConstBufferSequence,
-        class WriteHandler =
+        BOOST_BEAST_ASYNC_TPARAM2 WriteHandler =
```

Comment:
This should just be `class` since it appears in the docs and we dont want to teach `BOOST_BEAST_ASYNC_TPARAM2`